src/test/scala/nl.knaw.dans.easy.file2bag/FilesXmlSpec.scala

```diff
@@ -74,19 +107,19 @@ class FilesXmlSpec extends AnyFlatSpec with Matchers {
     (triedNode.get \ "file").last.serialize shouldBe
       """<?xml version='1.0' encoding='UTF-8'?>
         |<file filepath="data/blabla.txt">
-        |  <dc:format>text/plain</dc:format>
+        |  <dcterms:format>text/plain</dcterms:format>
         |</file>""".stripMargin
   }
 
-  it should "add a file element to files.xml, format node with 'dcterms' prefix" in {
+  it should "add a file element with dcterms prefix in format tag to an empty files.xml" in {
     val triedNode = FilesXml(
-      oldFilesXmlNoFileElements,
+      filesXmlNoFileElements,
       accessRights = "SOME",
       mimeType = "text/plain",
       destinationPath = Paths.get("blabla.txt"),
     )
     triedNode shouldBe a[Success[_]]
-    (triedNode.get \ "file").last.serialize shouldBe
+    (triedNode.get \ "file").head.serialize shouldBe
```

Comment:
please add the closing bracket yourself.
```suggestion
    (triedNode.get \ "file").headOption.map(_.serialize) shouldBe Some(
```

In general avoid plain head or last, there may not be one, then you get meaningless error messages.

Reply:
But in this test there should be a `file` element.  If not then we get `next on empty iterato`r error message.